mappings/net/minecraft/entity/mob/impl/EntityEnderman.mapping

```diff
@@ -1,8 +1,22 @@
 CLASS none/zg net/minecraft/entity/mob/impl/EntityEnderman
-	METHOD G getSoundDeath ()Lnone/nk;
+	FIELD bA ageWhenTargetSet I
+	FIELD bx CARRIED_BLOCK Lnone/kj;
+	FIELD by ANGRY Lnone/kj;
+	FIELD c CARRIABLE_BLOCKS Ljava/util/Set;
+	METHOD G getSoundAmbient ()Lnone/nk;
+	METHOD a setCarriedBlock (Lnone/asj;)V
+		ARG 0 value
 	METHOD a deserializeEntityTag (Lnone/dt;)V
 	METHOD a onTrackedDataSet (Lnone/kj;)V
+		ARG 0 data
 	METHOD a damage (Lnone/rp;F)Z
 		ARG 0 source
 		ARG 1 damage
 	METHOD b serializeEntityTag (Lnone/dt;)V
+	METHOD bV getSoundHurt ()Lnone/nk;
+	METHOD bW getSoundDeath ()Lnone/nk;
+	METHOD c setTarget (Lnone/sn;)V
+		ARG 0 value
+	METHOD dg getCarriedBlock ()Lnone/asj;
```

Comment:
Held is already used for item stacks that are being held in either hand and actually count as equipment.